src/test/java/com/mealmentor/enterprise/EnterpriseApplicationTests.java

```diff
@@ -4,7 +4,6 @@
 import com.mealmentor.enterprise.dto.*;
 import com.mealmentor.enterprise.service.IMealPlanService;
 import com.mealmentor.enterprise.service.MealPlanServiceStub;
-import lombok.SneakyThrows;
```

Comment:
Removed unused imports as part of the good clean coding conventions. 

To learn more: https://www.baeldung.com/java-clean-code